.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+on:
+  push:
+    branches: [develop, master]
+  pull_request:
+      types: [opened, synchronize, reopened]
+jobs:
+  build:  # Define a job named 'build'
+    defaults:  # Set default values for the job
+      run:
+        shell: bash
+        working-directory: "/opt/carma/"
+    runs-on: ubuntu-latest  # Run on the latest version of Ubuntu
+
+    container:  # Use a Docker container as the execution environment
+      image: ghcr.io/usdot-fhwa-stol/carma-builds-x64:focal  # Docker image to use
```

Comment:
is this going to work still? I think now we need carma-system-4.5.0 right?

Reply:
Actually I will make the change if only to guarantee a stable image